hail/python/hailtop/batch/backend.py

```diff
@@ -27,8 +27,8 @@
 from .globals import DEFAULT_SHELL
 
 
-HAIL_GENETICS_HAIL_IMAGE = os.environ.get('HAIL_GENETICS_HAIL_IMAGE',
-                                          f'hailgenetics/hail:{pip_version()}')
+HAIL_GENETICS_HAILTOP_SLIM_IMAGE = os.environ.get('HAIL_GENETICS_HAILTOP_SLIM_IMAGE',
+                                                  f'hailgenetics/hail:{pip_version()}')
```

Comment:
I remember we had an issue where, when I added `hailgenetics/hail` as the, I think, cleanup image, that broke the Australians. Can you dig that up in Zulip and make sure this change won't affect them? I think the issue might be related to how we handle the transaction from "public" images like `hailgenetics/hailtop-slim` to GCR images.

Reply:
They would indeed need to manually deploy this image into their GCR.